src/workerd/api/global-scope.c++

```diff
@@ -478,6 +494,7 @@ void ServiceWorkerGlobalScope::sendHibernatableWebSocketMessage(
 
 void ServiceWorkerGlobalScope::sendHibernatableWebSocketClose(
     HibernatableSocketParams::Close close,
+    kj::Maybe<int> eventTimeout,
```

Comment:
The user facing type is `unit32_t`, but here we have `int`, which we later verify is over 0. This means that half of our range is lost. We should make it a `unit32_t` here too.